Sources/TuistKit/Commands/CleanCommand.swift

```diff
@@ -10,12 +34,20 @@ struct CleanCommand: ParsableCommand {
         )
     }
 
-    @Argument(help: "The cache categories to be cleaned. If no category is specified, the whole cache is cleaned.")
-    var cacheCategories: [CacheCategory] = CacheCategory.allCases
+    @Argument(help: "The cache and artifact categories to be cleaned. If no category is specified, everything is cleaned.")
```

Comment:
Maybe we can add the categories to the help message by iterating through all the cases?

Reply:
if you run help, for this argument you will get `The cache and artifact categories to be cleaned. If no category is specified, everything is cleaned. (default: plugins, builds, tests, generatedAutomationProjects, projectDescriptionHelpers, manifests, dependencies)`, so it's already include :tada: